source/_integrations/modbus.markdown

```diff
@@ -339,10 +339,16 @@ modbus:
 
 modbus entities are grouped below each modbus communication entry.
 
-All modbus entities have the following parameters:
+**REMARK** Each modbus device must have at least 1 entity defined, otherwise the integration will not be loaded.
+
+Experience has shown that a modbus configuration without entities leads to instability issues.
+
+The normal use case for the integration is to have entities, however there are some special cases and custom components that used the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```

Comment:
```suggestion
The normal use case for the integration is to have entities, however there are some special cases and custom components that use the modbus integration without entities. The solution to these special cases is to add a dummy entity.
```

Reply:
Removed all the explaining, which really do not bring anything. The remark states clearly it is not allowed, and will prohibit the integration from loading.